File: Practice/CPP/2023-9-24-sort/SortUtils.cpp
#include"Head.h"

void BubbleSort(int *array,int length){
	
	// int length = GetArrayLength(array);


	for(int i = 0; i < length; i++){
		for(int j = 0; j < length - i - 1; j++){
			if(*(array + j) > *(array + j+1)){
				Swap(*(array+j),*(array+j+1));
			}
		}
	}
}

File: Practice/CS/2023-10-1-blog/Utils/SortUtils.cs
using System;
using Utils;
namespace SortUtils{
  public class SortUtils{
    public static void BubbleSort(int[] arr){
      for(int i = 0; i < arr.Length; i++){
        for(int j = 0; j < arr.Length - i - 1; j++){
          if(arr[j] > arr[j+1]) Utils.Utils.Swap(ref arr[j], ref arr[j+1]);
        }
      }
    }
  }
}
